source/common/grpc/common.cc

```diff
@@ -53,8 +53,8 @@ void Common::chargeStat(const Upstream::ClusterInfo& cluster, const std::string&
   chargeStat(cluster, "grpc", grpc_service, grpc_method, success);
 }
 
-Optional<Status::GrpcStatus> Common::getGrpcStatus(const Http::HeaderMap& trailers) {
```

Comment:
I think it's complaining that the .cc definition doesn't match the .h definition. Do you mean it was intentional in this case?